components/filters.tsx

```
@@ -580,6 +595,46 @@ export function Tags(props: { tags: Tag[] }) {
+                ? "bg-primary-500 text-white border-primary-500 shadow-sm"
+                : "border border-dark--theme-200 dark:border-dark--theme-800",
+              "hover:shadow-md hover:border-primary-200 dark:hover:border-primary-800"
+            )}
+          >
+            {!isAnyTagActive && (
+              <svg
+                className="w-3 h-3 mr-1.5 text-white"
+                fill="none"
+                stroke="currentColor"
+                viewBox="0 0 24 24"
+              >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)